app/controllers/pairings_controller.rb

```diff
@@ -18,7 +18,18 @@ def index
         player2_name: p.player1.name_with_pronouns,
         pairing: p
       }
-    end.sort_by { |p| p[:player1_name] }
+    end.sort_by do |p|
+      # Sort by username for doublesided, but Corp username for Single-Sided Swiss.
+      if p[:pairing].round.stage.format == :single_sided_swiss.to_s
+        if p[:pairing].side == 'player1_is_corp'
```

Comment:
I think this could call `p[:pairing].player1_is_corp?`.

Reply:
no longer needed.